docs/services/alertmanager.md

```diff
@@ -0,0 +1,106 @@
+# Alertmanager
+
+## Parameters
```

Comment:
Would you write whether default or not as follows, please?
https://github.com/argoproj/notifications-engine/blob/master/docs/services/slack.md#parameters